besu/src/test/java/org/hyperledger/besu/util/LocalPermissioningConfigurationValidatorTest.java

```diff
@@ -123,4 +136,89 @@ public void nodeAllowlistCheckShouldIgnoreDiscoveryPortParam() throws Exception
           "Exception not expected. Validation of nodes in allowlist should ignore the optional discovery port param.");
     }
   }
+
+  @Test
+  public void nodeAllowlistCheckShouldWorkWithHostnameIfDndEnabled() throws Exception {
```

Comment:
Is there a typo in this test name? (`Dnd` instead of `Dns`)

Reply:
Good catch . Changed